mlir/lib/Catalyst/Transforms/catalyst_to_llvm.cpp

```diff
@@ -444,7 +444,7 @@ struct PythonCallOpPattern : public OpConversionPattern<PythonCallOp> {
 
         bool isVarArg = true;
         LLVM::LLVMFuncOp customCallFnOp = mlir::LLVM::lookupOrCreateFn(
-            mod, "pyregistry", {/*args=*/i64, i64, i64}, /*ret_type=*/voidType, isVarArg);
+            mod, "inactive_callback", {/*args=*/i64, i64, i64}, /*ret_type=*/voidType, isVarArg);
```

Comment:
Would you mind just describing how inactive_callback could be a replacement for pyregistry (on a logical level), they seem somewhat like disjoint concepts (although it looks like this is purely a name change, but it would help me understand how to think about it).